spec/abstract-operations.html

```diff
@@ -617,54 +617,76 @@ <h1>RequireObjectCoercible ( _argument_ )</h1>
         </table>
       </emu-table>
     </emu-clause>
+  </emu-clause>
+
+  <emu-clause id="sec-samevalue" aoid="SameValue">
+      <h1>SameValue ( _x_, _y_ )</h1>
+      <p>The abstract operation SameValue takes arguments _x_ (an ECMAScript language value) and _y_ (an ECMAScript language value) and returns a completion record whose [[Type]] is ~normal~ and whose [[Value]] is a Boolean. It performs the following steps when called:</p>
+      <emu-alg>
+        1. If Type(_x_) is different from Type(_y_), return *false*.
+        1. If Type(_x_) is <del>Number or BigInt</del><ins>Number, BigInt, Record, or Tuple</ins>, then
+          1. Return ! Type(_x_)::sameValue(_x_, _y_).
+        1. Return ! SameValueNonNumeric(_x_, _y_).
```

Comment:
SameValueNonNumeric seems like it’ll need a rename now

Reply:
I agree, I couldn't think of a better name so I left it the same.

Reply:
SameValueNonNumericOrRecordOrTuple ? 🤣

Reply:
`SameValueNonGeneric`? since the things that are exempted from it are using the generic-ish T::foo() approach?